.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Feature
+about: Issues for requesting feature changes or additions.
+title: ''
+labels: 'status: planning, type: feature'
+assignees: ''
+
+---
+
+## Description
+<!-- Describe your proposal in detail -->
+
+
+## Reasoning
+<!-- Outline how the proposal would be useful to the community -->
+
+
+## Proposed Implementation
+<!-- Detail any implementation you had in mind, including any screenshots if possible -->
+
+
+## Additional Details
+<!-- If alternate implementations were considered, add them here -->
+<!-- Delete this section if there are no details to add -->
+
+
+## Would you like to reserve implementing it?
```

Comment:
```suggestion
## Would you like to implement this yourself?
```
Using the word `reserve` here seems like sacrificing readability with younger users, especially with English as a second language.